hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -292,13 +321,23 @@ export class DialogHassioNetwork extends LitElement implements HassDialog {
       | null
       | undefined
       | string[] = (ev.target as PaperInputElement).value;
+
     if (!value || !this._device) {
       return;
     }
+
     const id = (ev.target as PaperInputElement).id;
```

Comment:
```suggestion
```

Reply:
?

Reply:
You should combine them all 😄

Reply:
Can't, it will break it

Reply:
Just check if the value that was changed was changed, instead of comparing the complete object